doc/dart_documentation_comment_specification.md

```diff
@@ -0,0 +1,478 @@
+# **Dart Documentation Comments Specification**
+Version: 1.0.0
+
+[TOC]
+
+## **1\. Introduction**
+
+### **1.1. Purpose**
+
+The purpose of this document is to provide a single specification for documentation comments in the Dart programming language. Its goal is to ensure that documentation is parsed consistently by all tools and to serve as a definitive standard for developers and tool authors.
+
+### **1.2. Scope**
+
+**In Scope**
+
+This specification covers the following topics:
+
+* The *syntax* for writing documentation comments.
+* The rules governing where documentation comments can be *placed*.
+* The mechanism for *resolving* references between elements.
+* The logic for *associating* documentation with code elements.
+
+**Out of Scope**
+
+This document does not cover:
+
+* The implementation details of specific documentation tools (e.g., `dartdoc`, `analysis server`).
+* The syntax and behavior of tool-specific directives (e.g., `@template`, `@canonicalFor`).
```

Comment:
Would you consider these in-scope in a later version? It would be very beneficial for DAS and dartdoc to agree on how these function. And new ones can be specified like `@snippet`, `@sample`.